### PR TITLE
Gemfile: Always use require: false

### PR DESCRIPTION
**Pull request by jayvdb**
**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by Nosferatul** [comment]
ack e10c844

**Comment by Nosferatul** [comment]
needs rebase

File: .ci/generate_bear_requirements.py [hunk 1]
```diff
@@ -126,8 +126,9 @@ def write_gem_requirements(requirements):
     for requirement in requirements:
         gem_dependencies.append(
             {'name': requirement.package,
-             'version': requirement.version,
-             'required': requirement.require})
```
Comment (satwikkansal) [inline .ci/generate_bear_requirements.py]: Can't we keep it and default it to `false`.

Comment (satwikkansal) [inline .ci/generate_bear_requirements.py]: Also, the template file `Gemfile.jinja2` needs to be updated if the `required` key is not used.

Comment (jayvdb) [inline .ci/generate_bear_requirements.py]: I'm still uncertain about this. This flag is recording useful information, just not very useful for our purposes.
So i think we keep the flag, in the Requirement.
I'd like to check if removing it from Gemfile improves performance at all.

**Comment by supergr35** [comment]
rebase please

**Comment by jayvdb** [comment]
reack https://github.com/coala/coala-bears/pull/1623/commits/803db03cb7553078f249cd94f85f80625c47d379

**Comment by jayvdb** [comment]
@rultor merge

**Comment by Makman2** [comment]
@gitmate-bot rebase

**Pull request merged**